### ISSUE TITLE
UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch

### ISSUE THREAD

**Issue by mbarnig** [mbarnig/Marylux-648-TTS-Corpus#5]
`__floordiv__` rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').

Some TTS models generate negative loss values during training with the Marylux-648 dataset. This can have an impact on the results.
